src/indev/lv_indev.h

```diff
@@ -113,31 +117,81 @@ lv_indev_t * lv_indev_active(void);
  */
 void lv_indev_set_type(lv_indev_t * indev, lv_indev_type_t indev_type);
 
+/**
+ * Set a function pointer to read input device data to the indev
```

Comment:
The better name instead of "funcion pointer": "callback function".
_Set a callback function to read input data from indev_

Reply:
Done in 25f295f